lib/pages/media_view.dart

```diff
@@ -19,6 +20,10 @@ class MediaViewPage extends HookWidget {
     final showButtons = useState(true);
     final isZoomedOut = useState(true);
 
+    final isDragging = useState(false);
+    final initialPositionY = useState<double>(0);
```

Comment:
Not needed, you can get the update the position with `positionYDelta.value += details.delta.dy;`